vertical-pod-autoscaler/FAQ.md

```diff
@@ -162,6 +163,16 @@ spec:
 
 and then deploy it manually if your vpa is already configured.
 
+### Can I run the VPA in an HA configuration?
+
+The VPA admission-controller can be run with multiple active Pods at any given time.
+
+Both the updater and recommender can only run a single active Pod at a time. Should you
+want to run a Deployment with more than one pod, it's recommended to enable a lease
+election with the `--leader-elect=true` parameter.
+
+**NOTE: If using GKE, you must set `--leader-elect-resource-name` to something OTHER than "vpa-recommender", for example "vpa-recommender-lease".**
```

Comment:
```suggestion
**NOTE:** If using GKE, you must set `--leader-elect-resource-name` to something OTHER than "vpa-recommender", for example "vpa-recommender-lease".
```

Reply:
@adrianmoisey, It is the last one, please correct the Note (only the Note should be in bold).
and also squash your all commits into one commit.

Reply:
Done, thanks.